readme_pages/SOURCEBUILD.md

```diff
@@ -233,11 +233,6 @@ Run either the C++ or the Python tests with datasets
 
 Note: This conda installation only applies to Linux and Python versions 3.8/3.10.
 
-### Building and Testing on a gpuCI image locally
```

Comment:
Should we provide updated instructions on this?  With the new GA approach, there are new scripts that provide this behavior.  Is there documentation somewhere that describes how one would run the process that mimics CI that we could link to here?

Reply:
You're ahead of the curve, @ChuckHastings.

I still have to get that new functionality properly documented somewhere.

I'll try to get to it this week.

Reply:
One of my life curses :-). Always a challenge when you live ahead of the curve.

Reply:
Opened a PR for these new docs below.

- https://github.com/rapidsai/docs/pull/335

Reply:
@ChuckHastings, with that new documentation almost published, would you mind approving this PR so we can get this merged?